src/Graphics/OpenGLContext/opengl_BufferManipulationObjectFactory.cpp

```diff
@@ -357,34 +363,44 @@ class BlitFramebuffersImpl : public BlitFramebuffers
 
 	BlitFramebuffersImpl(CachedBindFramebuffer * _bind,
 		CachedEnable * _enableScissor,
-		Renderer _renderer)
+		Renderer _renderer, bool _copyImage)
 		: m_bind(_bind)
 		, m_enableScissor(_enableScissor)
-		, m_renderer(_renderer) {
+		, m_renderer(_renderer)
+		, m_copyImage(_copyImage) {
 	}
 
 	bool blitFramebuffers(const graphics::Context::BlitFramebuffersParams & _params) override
 	{
-		m_bind->bind(graphics::bufferTarget::READ_FRAMEBUFFER, _params.readBuffer);
-		m_bind->bind(graphics::bufferTarget::DRAW_FRAMEBUFFER, _params.drawBuffer);
-
-		const s32 adrenoCoordFix = (m_renderer == Renderer::Adreno) ? 1 : 0;
-
-		m_enableScissor->enable(false);
-		glBlitFramebuffer(
-			_params.srcX0, _params.srcY0, _params.srcX1, _params.srcY1,
-			adrenoCoordFix +_params.dstX0, _params.dstY0, _params.dstX1, _params.dstY1,
-			GLbitfield(_params.mask), GLenum(_params.filter)
-		);
-		m_enableScissor->enable(true);
-
+		if (_params.dstX1 - _params.dstX0 == _params.srcX1 - _params.srcX0 &&_params.dstY1 - _params.dstY0 == _params.srcY1 - _params.srcY0 &&
+		_params.readBufferAttachment != graphics::ObjectHandle() && _params.drawBufferAttachment != graphics::ObjectHandle() && m_copyImage) {
```

Comment:
m_copyImage need to be checked first: why check parameters if m_copyImage is not supported?